src/ResourceManager/IotHub/Commands.IotHub/packages.config

```diff
@@ -1,4 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.Management.IotHub" version="1.1.2" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.IotHub" version="1.1.3" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
remove these three packages - they are imported with common dependencies and so are not necessary here.

Reply:
Removed